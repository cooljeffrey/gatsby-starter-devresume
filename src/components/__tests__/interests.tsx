import React from "react";
import renderer from "react-test-renderer";
import Interests from "../interests";
describe("Educations", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Interests items={["Hobby", "Another Hobby"]} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
