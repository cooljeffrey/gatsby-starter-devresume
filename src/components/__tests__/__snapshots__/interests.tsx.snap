// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Educations renders correctly 1`] = `
<section
  className="skills-section py-3"
>
  <h3
    className="text-uppercase resume-section-heading mb-4"
  >
    Interests
  </h3>
  <ul
    className="list-unstyled resume-interests-list mb-0"
  >
    <li
      className="mb-2"
    >
      Hobby
    </li>
    <li
      className="mb-2"
    >
      Another Hobby
    </li>
  </ul>
</section>
`;
